CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 ### Bug fixes
 
+* [#9232](https://github.com/rubocop-hq/rubocop/pull/9232): Fix `Style/SymbolProc` registering wrong offense when using a symbol numbered block argument greater than 1, i.e. `[[1, 2]].map { _2.succ }`. ([@tdeo][])
```

Comment:
You misplaced this changelog entry. Now they go under the `changelog` dir and this file is auto-generated on release.

Reply:
Indeed, read the checklist a bit too fast and assumed it still was the same, I amended the commit